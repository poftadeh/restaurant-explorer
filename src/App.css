*,
*::before,
*::after {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

html {
  font-size: 62.5%;
}

html,
body {
  height: 100%;
}

body {
  font-family: Roboto, Arial, Helvetica, sans-serif;
  line-height: 1.7;
  background-color: #b2b7ff;
  display: flex;
  justify-content: center;
  align-items: center;
}

.results {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  width: 100%;
  margin-top: 2rem;
  height: 50rem;
}
