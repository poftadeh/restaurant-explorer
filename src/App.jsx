import React from 'react';
import 'fontsource-roboto';
import { Provider } from 'react-redux';
import store from './store';
import './App.css';
import Container from './components/Container';
import Search from './components/Search';
import Heading from './components/Heading';
import RestaurantTable from './components/RestaurantTable';

const App = () => (
  <Provider store={store}>
    <Container>
      <Heading />
      <Search />
      <RestaurantTable />
    </Container>
  </Provider>
);

export default App;
