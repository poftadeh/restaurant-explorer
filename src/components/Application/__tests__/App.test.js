import React from 'react';
import { shallow } from 'enzyme';
import App from '..';

it('renders <App /> properly', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
