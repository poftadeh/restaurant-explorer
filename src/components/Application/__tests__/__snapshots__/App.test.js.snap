// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <App /> properly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Container>
    <Heading />
    <Connect(Search) />
    <section
      className="results"
    >
      <Connect(RestaurantTable) />
      <Connect(Status) />
    </section>
  </Container>
</Provider>
`;
